crates/transaction-pool/src/noop.rs

```diff
@@ -42,7 +42,7 @@ impl<T: EthPoolTransaction> NoopTransactionPool<T> {
     }
 }
 
-impl Default for NoopTransactionPool<EthPooledTransaction> {
+impl<T> Default for NoopTransactionPool<T> {
```

Comment:
let's undo this, this is worse ux for default use case

Reply:
hm but it's needed for OP too in order to construct `NoopTransactionPool::default` in context of op-reth